### ISSUE TITLE
Aerial Interface not charging air gear

### ISSUE THREAD

**Issue by Brettflan** [FTBTeam/FTB-Modpack-Issues#4269]
### Modpack

FTB Genesis

### Modpack version

1.2.0

### Describe the bug

I'm not familiar enough with Pneumaticcraft to know for sure that this is a bug, but it sure seems like it. An internet search gave me the impression this should work. The Aerial Interface is not charging pressure into my jackhammer or wrench. I'm not wearing Pneumatic armor, if that is for some reason needed.

I crafted the Jackhammer for that quest, and it implied the Aerial Interface would keep it charged.

Oddly enough, it is in fact working to charge my FE/AE powered gear from other mods. Just not the air-pressure tools from the mod itself. I do have just under 20 bars of pressure in the device, as can be seen in the screenshot.

### Steps to reproduce

1. Place a Pneumaticcraft Aerial Interface and provide it with >10 bar air pressure and power.
2. Confirm it indicates it is linked to your player in the "Problems" popout in the interface for it.
3. Have a not fully charged Pneumaticcraft tool such as a wrench or jackhammer in your inventory, as well as something that takes FE or AE power or similar.
4. Note that the FE/AE gear gets charged by the Aerial Interface, but the wrench/jackhammer/whatever does not.

### Expected behaviour

For the Aerial Interface to remotely charge Pneumaticcraft gear.

### Screenshots

![aerial_interface_not_charging](https://github.com/FTBTeam/FTB-Modpack-Issues/assets/592130/3b57b36d-d8a8-4714-91f1-a70a7a999416)

### Additional information

I added the Dispenser module just to see if it fixed it; it did not. It doesn't work with or without that module installed.

**Comment by Brettflan** [comment]
Also, while the jackhammer is showing 10.0 bar, it is not actually fully charged, as can be seen by the existence of a bar underneath it. I just had it nearly charged up fully at that point. I since ran it down to 9.8, where it is currently staying.

**Comment by Brettflan** [comment]
And I've now crafted and equipped the full set of Pneumaticcraft armor, and added the quest provided modules for charging and volume (and Elytra), and... the Aerial Interface is not charging them either.

**Comment by Brettflan** [comment]
Something I probably should have checked already was the Pneumaticcraft manual in the tome, it does have further info on the Aerial Interface. It indicates a Charging Station needs to be pointed at the side of it to charge items, and that side configuration needs to be set appropriately for what you want it to charge.

However, that also didn't work for me. I tried facing a pressurized Charging Station at one side and switched the side config between armor and main inventory, but nothing got charged. I then rotated it around so it was connected to the Aerial Interface for air supply, and tried both side configs. No luck with that either.

I did find that the auto-charging functionality of the armor does work, when the chestplate has higher pressure than my tools, it charges them up. It's just the Aerial Interface that isn't providing air to anything on me.

**Comment by Saereth** [comment]
seems to be working fine for me? Can you show a screenshot of your setup?
![2023-11-18_11 00 47](https://github.com/FTBTeam/FTB-Modpack-Issues/assets/9200909/038ac077-f588-4de9-bd52-f1e1dd896ebd)

![2023-11-18_11 00 42](https://github.com/FTBTeam/FTB-Modpack-Issues/assets/9200909/b6ea4b87-834b-4bc6-9e90-d416ffc74fbd)

**Comment by Brettflan** [comment]
Aha, it was user error! Mainly due to my lack of knowledge of the mod. In the ingame book for the mod it mentions the "charging module", which I skimmed over and mistakenly read as "charging station". I didn't even realize that part was a thing. Admittedly that's the only place I recall seeing explicit info on how to set it up, all other references I got from a quick search were vague and seemed to generally imply it would just work without any additional components.

Anyway, now that I have that module installed and aimed at the aerial interface, it is indeed working. Thanks for the screenshot, it tipped me off.

Might be good to have some info somewhere in the questline mentioning it.
